gradle.properties

```diff
@@ -2,6 +2,6 @@
 fiatVersion=1.12.0
 enablePublishing=false
 spinnakerGradleVersion=7.0.1
-korkVersion=7.7.0
+korkVersion=7.4.0-SNAPSHOT
```

Comment:
Looks like an artifact from your local testing.

Reply:
Yes. Since the kork PR this depends on isn't merged, there's no need for me to update this yet.